README.md

```diff
@@ -26,27 +25,27 @@ Requires Node.js 10+
 To install via NPM:
 
 ```
-npm install context-template
+npm install ed25519-signature-2020-context
 ```
 
 ## Usage
 
 ```js
 const {
   contexts, constants, appContextMap, documentLoader
-} = require('context-template');
+} = require('ed25519-signature-2020-context');
 
 // use URL in a JSON-LD context
 const obj = {
   "@context": [
-    constants.EXAMPLE_CONTEXT_URL,
+    constants.CONTEXT_URL,
     // ...
   ],
   // ...
 };
 
 // get context data for a specific context
-const data = contexts.get('https://w3id.org/example/v1');
+const data = contexts.get('https://w3id.org/security/ed25519-signature-2020/v1');
```

Comment:
```suggestion
const data = contexts.get('https://w3id.org/security/suites/ed25519-2020/v1');
```

Just a suggestion... thoughts?

Reply:
@msporny I like `'https://w3id.org/security/suites/ed25519-2020/v1'`. Changing it to that unless @dlongley objects.